.vib/mongodb/goss/goss.yaml

```diff
@@ -1,33 +1,36 @@
-process:
-  mongod:
-    running: true
+file:
+  /bitnami/mongodb:
+    exists: true
+    filetype: directory
+    mode: "2775"
+    owner: root
+  /opt/bitnami/mongodb/conf/mongodb.conf:
+    exists: true
+    filetype: file
+    mode: "0660"
+    contains:
+      - /port:.*{{ .Vars.containerPorts.mongodb }}/
+  /var/run/secrets/kubernetes.io/serviceaccount:
+    exists: {{ .Vars.serviceAccount.automountServiceAccountToken }}
+    filetype: directory
+    mode: "3777"
 command:
-  mongod-version:
-    exec: mongod --version | grep --quiet "$APP_VERSION"
-    exit-status: 0
-    stdout: [ ]
-    stderr: [ ]
-    timeout: 20000
+  {{- $conn_opts := printf "--quiet --username %s --password '%s' --host mongodb --port %d" .Vars.auth.rootUser .Vars.auth.rootPassword .Vars.service.ports.mongodb }}
   admin-command:
-    exec: mongosh --quiet --username "$MONGODB_ROOT_USER" --password "$MONGODB_ROOT_PASSWORD" --port "$MONGODB_PORT_NUMBER" --eval "db.adminCommand({listCollections:1})"
+    exec: mongosh {{ $conn_opts }} --eval "db.adminCommand({listCollections:1})"
     exit-status: 0
-    stdout:
-      - "system.version"
-    stderr: [ ]
-    timeout: 20000
+    timeout: 5000
```

Comment:
IIRC, the default timeout is 10s so maybe we can remove this line if reducing it is not a must. Same to with the test below.

Reply:
Yeah, thanks. Don't really know why I added it tbh.